game/tools/subtitles/subtitle_editor.cpp

```diff
@@ -306,7 +332,7 @@ void SubtitleEditor::draw_repl_options() {
 
 void SubtitleEditor::draw_all_cutscene_groups() {
   for (auto& group_name : m_subtitle_db.m_subtitle_groups->m_group_order) {
-    ImGui::SetNextItemOpen(true);
+    // ImGui::SetNextItemOpen(true);
```

Comment:
maybe we could have an "open all" checkbox or something here, yeah. I used to use the filter box sometimes before my changes, and now not really at all, I agree I made it less useful in doing this

Reply:
well as i mention, you could just change the `true` to `!m_filter.empty() || !m_filter_hints.empty()` and it might work out great.

Reply:
I didn't put that check as an argument to the call since that would force the groups closed if empty or placeholder string was found, but I added this check